nncf/quantization/algorithms/min_max/torch_fx_backend.py

```diff
@@ -56,13 +56,15 @@
 
 class FXMinMaxAlgoBackend(MinMaxAlgoBackend):
 
+    MAT_MUL_METATYPES = [om.PTLinearMetatype, om.PTMatMulMetatype]
+
     @property
     def preserved_metatypes(self) -> List[OperatorMetatype]:
         return []
 
     @property
     def mat_mul_metatypes(self) -> List[OperatorMetatype]:
-        return [om.PTLinearMetatype, om.PTMatMulMetatype]
+        return self.MAT_MUL_METATYPES
```

Comment:
Do you introduce second way to get access to list of metatypes, in backends it will be used as self.MAT_MUL_METATYPE in algorithm.py `backend.mat_mul_metatypes`?

It is not good, please keep only one way to access to mat_mul_metatypes.

Reply:
Ok, then it will be `PTMinMaxBackend().mat_mul_metatypes`. Are you ok with that?